sample/OSCON17/event

```diff
@@ -0,0 +1,78 @@
+{
+	"id": 3,
+	"name": "OSCON open source convention",
+	"latitude": 30.263569,
+	"longitude": -97.739606,
+	"location_name": "Austin Convention Center, Austin, TX ",
+	"start_time": "2017-05-08T13:00:00",
+	"end_time": "2017-05-11T20:00:00",
+	"timezone": "UTC",
```

Comment:
@mahikaw the timings of the event are according to the local place. so we are not focusing on what timezone is international etc but the timezone where the event takes place.